doc/source/development/contributing.rst

```diff
@@ -1610,6 +1610,28 @@ request by pushing to the branch on GitHub::
 
     git push origin shiny-new-feature
 
+Autofixing formatting errors
+----------------------------
+
+We use several styling checks(e.g. ``black``, ``flake8``, ``isort``) which are run after
+you make a pull request. If there is a scenario where any of these checks fail then you
+can comment::
+
+    @githubactions pre-commit
+
+on that pull request. This will trigger a workflow which will autofix formatting errors.
+
+Backporting (optional)
```

Comment:
this should only be in the maintainers section (i think we have one?)

Reply:
Yes, doc/source/development/maintaining.rst - @1nF0rmed could you move this over there please?

Reply:
Does that include the backporting section?

Reply:
yes, the whole part about `@meeseeksdev backport`